crates/ursa-rpc-service/src/api.rs

```diff
@@ -216,6 +216,18 @@ where
 
         info!("The inserted cids are: {cids:?}");
 
+        let (sender, receiver) = oneshot::channel();
+        let request = NetworkCommand::Put {
+            cid: root_cid,
+            sender,
+        };
+        self.network_send.send(request)?;
+        if let Err(e) = receiver.await {
+            return Err(anyhow!(format!(
+                "The PUT failed, please check server logs {e:?}"
+            )));
```

Comment:
just a minor one, for shorten.
```suggestion
            bail!("The PUT failed, please check server logs {e:?}");

```